code/API_definitions/location.yaml

```
@@ -200,6 +250,92 @@ components:
         message:
           type: string
           description: Detailed error description
+    LocationEventType:
+      description: |
+        AREA_ENTERED - Location event triggered when the UE entered the given area
+        AREA_LEFT - Location event triggered when the UE left the given area
+      type: string
+      enum:
+        - AREA_ENTERED
+        - AREA_LEFT
+    LocationEventTypeList:
+      description: List of Location Event Type
+      type: array
+      items:
+        $ref: '#/components/schemas/LocationEventType'
+      example: [ AREA_ENTERED, AREA_LEFT ]
+    CreateSubscription:
+      description: The request for creating a Location Subscription
+      type: object
+      properties:
+        ueId:
```

```suggestion
        ueIds:
```
If a list we should use a plural, or ueIdList